build.sbt

```diff
@@ -61,7 +61,10 @@ lazy val `kyo-settings` = Seq(
     Test / testOptions += Tests.Argument("-oDG"),
     ThisBuild / versionScheme               := Some("early-semver"),
     libraryDependencies += "org.scalatest" %%% "scalatest" % scalaTestVersion % Test,
-    Test / javaOptions += "--add-opens=java.base/java.lang=ALL-UNNAMED"
+    Test / javaOptions += "--add-opens=java.base/java.lang=ALL-UNNAMED",
+    javaOptions ++= Seq(
+        "-Xmx12G"
```

Comment:
This is quite a lot of heap, please revert it. Can you share the failure that led to this change?